Update description of the config `server.cypher.parallel.worker_limit` (#1104)
neo4j/docs-operations: modules/ROOT/pages/configuration/configuration-settings.adoc

If set to a positive number, that number of workers will be started.
If set to `0`, one worker will be started for every logical processor available to the Java Virtual Machine.

> If set to a negative number, that number will be subtracted from the total number of logical processors available on the server.

Reviewer: ```suggestion
If set to a negative number, the total number of logical processors available on the server will be reduced by the absolute value of that number.
```
